resources/mac/template.app/Contents/Info.plist

```diff
@@ -1 +1,90 @@
-// Info.plist should be put here
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
What are `Info.plist` and `document.wflow`? Are these files documented somewhere online? There is a lot of content in these files, like version numbers and cryptic information.

Reply:
There is an application called `Automator` which can create application on MacOS. 
I created an application by this `Automator` which can run script.
Then I replaced the script in order to start our apps.

Info.plist and document.wflow are two files included when `Automator` generated the `something.app`.

Info.plist contains the configuration of icon path and app identifier.
document.wflow contains the script.

I knew there are unnecessary content inside these two files,
but since it takes sometime to do research on it,
I decided to make it work first.